client/dashboard/sites/settings-mcp/setup.tsx

```diff
@@ -33,19 +33,17 @@ function McpSetupComponent( { siteSlug }: { siteSlug: string } ) {
 		{ label: 'VS Code', value: 'vscode' },
 		{ label: 'Cursor', value: 'cursor' },
 		{ label: 'Continue', value: 'continue' },
-		{ label: 'Llamafile', value: 'llamafile' },
 	];
 
 	// Documentation links for each client
 	const clientDocumentation = {
-		claude: 'https://docs.anthropic.com/en/docs/claude-desktop-mcp',
+		claude: 'https://docs.claude.com/en/docs/mcp',
 		vscode: 'https://code.visualstudio.com/docs/copilot/customization/mcp-servers',
 		cursor: 'https://docs.cursor.com/en/context/mcp',
 		continue: 'https://docs.continue.dev/customize/deep-dives/mcp',
-		llamafile: 'https://github.com/Mozilla-Ocho/llamafile',
 		default: 'https://modelcontextprotocol.io/docs/servers',
 	};
-	const serverName = `${ site.slug }-mcp`;
+	const serverName = `${ site?.slug }-mcp`;
```

Comment:
We have used `useSuspenseQuery` above so the site won't be null 🤔

Reply:
Ok, I'll remove it, thanks